src/build.py

```diff
@@ -1397,7 +1406,7 @@ def TestBare():
       wasmjs=os.path.join(INSTALL_LIB, 'wasm.js'),
       extra_files=[os.path.join(INSTALL_LIB, 'musl.wasm')])
 
-  if IsMac():
+  if IsMac() and 'JSC' not in buildbot.WarnedList():
```

Comment:
This is a step in the right direction, and we should keep it for this PR. But we might want to do the same thing for other tests, e.g. the bare tests depend on the musl build, and the emscripten tests depend on the emscripten build. So in the future we might consider some more generic dpendence mechanism.